WINDOWS.md

```diff
@@ -0,0 +1,56 @@
+# Cage on Windows
+
+Generally speaking, docker-compose and docker engine are a work in progress on
+Windows. This document should give you all the information you need on
+available builds and overall guidance when working with cage
+on Windows.
+
+## Setup
+
+First make sure you install rust for x86_64-pc-windows-gnu using the
+`rustup-init` installer from [https://www.rustup.rs/](https://www.rustup.rs/).
+You should make sure you use either `stable` or `beta` toolchains.
```

Comment:
I think we can just recommend `stable` here. `beta` is basically just for CI systems to check against the next `stable`.